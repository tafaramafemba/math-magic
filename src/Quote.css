.quote {
  width: 50%;
  margin: auto;
  padding-top: 60px;
}
