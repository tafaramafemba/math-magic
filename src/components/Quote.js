import '../Quote.css';

function Quote() {
  return (
    <div>
  <h3 className='quote'>Mathematics is not about numbers, equations, computations,
    or algorithms: it is about understanding. - William Paul Thurston
  </h3>
    </div>
  );
}

export default Quote;