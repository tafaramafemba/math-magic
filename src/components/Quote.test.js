import renderer from 'react-test-renderer'
import Quote from "./Quote";

describe(' Quote Component Rendering ', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Quote />)
    expect(tree).toMatchSnapshot();
  });
})