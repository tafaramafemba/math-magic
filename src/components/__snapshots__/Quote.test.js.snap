// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Home Component Rendering  renders correctly 1`] = `
<div>
  <h3
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </h3>
</div>
`;
